resources/manifests/psp-restricted.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: restricted
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+spec:
+  privileged: false
+  # Required to prevent escalations to root.
+  allowPrivilegeEscalation: false
+  # This is redundant with non-root + disallow privilege escalation,
+  # but we can provide it for defense in depth.
+  requiredDropCapabilities:
+  - KILL
+  - MKNOD
+  - SETUID
+  - SETGID
+  # Allow core volume types.
+  volumes:
+  - 'configMap'
+  - 'emptyDir'
+  - 'projected'
+  - 'secret'
+  - 'downwardAPI'
+  # Assume that persistentVolumes set up by the cluster admin are safe to use.
+  - 'persistentVolumeClaim'
+  hostNetwork: false
+  hostIPC: false
+  hostPID: false
+  runAsUser:
+    # Require the container to run without root privileges.
+    rule: 'MustRunAs'
+    ranges:
+    - min: 10000
```

Comment:
What's the reasoning behind `min 10000` ? Containers use for example UID 33 (www-data), which would not start in this configuration if I understand correctly.

Reply:
Is `33` a fixed UID for www-data?

Reply:
I only know that it's the default on Debian. I would suggest we use `min: 1` unless we have good reason to not do that (and we can always adjust that later, when we have a clearer idea of our requirements).

Reply:
ok gotcha.